tools/ProjectTemplates/dev.cpp.dll.winrt-client-server/PurojekutoTenpuret.idl

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+namespace Microsoft.PurojekutoTenpuret
+{
+    [contractversion(1)]
+    apicontract PurojekutoTenpuretContract{};
+
+    [contract(PurojekutoTenpuretContract, 1)]
+    runtimeclass AppGraphManager
```

Comment:
BUG! Should be `runtimeclass PurojekutoTenpuretManager`

Fix rest of IDL file too